source/updateCheck.py

```diff
@@ -369,9 +369,11 @@ def __init__(self, parent, updateInfo: Optional[Dict], auto: bool) -> None:
 			# Translators: A message indicating that no update to NVDA is available.
 			message = _("No update available.")
 		elif canOfferPendingUpdate:
-			# Translators: A message indicating that an updated version of NVDA has been downloaded
-			# and is pending to be installed.
-			message = _("NVDA version {version} has been downloaded and is pending installation.").format(**updateInfo)
+			message = _(
+				# Translators: A message indicating that a new version of NVDA has been downloaded and is ready to be

```

Comment:
On the theory that one doesn't apply a version, one applies an update.

```suggestion
				# Translators: A message indicating that an update to NVDA has been downloaded and is ready to be
```